serverlessworkflow/modules/ROOT/pages/use-cases/timeout-showcase-example.adoc

```diff
@@ -10,10 +10,38 @@ While all the scenarios contain the same set of workflows, they are provided as
 
 The following workflows are provided:
 
+* <<workflow_timeouts_workflow, `workflow_timeouts`>>
 * <<callback_state_timeouts_workflow, `callback_state_timeouts`>>
 * <<switch_state_timeouts_workflow, `switch_state_timeouts`>>
 * <<event_state_timeouts_workflow, `event_state_timeouts`>>
 
+[#workflow_timeouts_workflow]
+== `workflow_timeouts` workflow
+
+It is a simple workflow that, rather than configuring a timeout for a particular state, it is configured for the hole execution of the workflow.
```

Comment:
```suggestion
It is a simple workflow that, rather than configuring a timeout for a particular state, is configured for the whole execution of the workflow.
```